src/FacebookEventForwarder.js

```diff
@@ -90,9 +98,15 @@
                                 visitorData['external_id'] = selectedIdentity[0].Identity;
                             }
                         }
+                    }
 
-                        fbq('init', settings.pixelId, visitorData);
+                    if (settings.disablePushState === 'True') {
+                        // Facebook will automatically track page views whenever a new state is pushed to the HTML 5 History State API
+                        // this option can be disabled to prevent duplicate page views
+                        // https://developers.facebook.com/docs/facebook-pixel/implementation/tag_spa/#tagging-single-page-applications
+                        fbq.disablePushState = true;

```

Comment:
Also, above, this is a string, and here it's a bool?

Reply:
Correct. The above is a string because the server sends it to us this way. Then since `fbq.disablePushState` is a Facebook API, it accepts a boolean.

Reply:
I feel like these should be two different attributes but what do I know?